src/tz_android.rs

```diff
@@ -1,12 +1,16 @@
+use std::ffi::CStr;
+
 use android_system_properties::AndroidSystemProperties;
+use once_cell::sync::OnceCell;
 
 // From https://android.googlesource.com/platform/ndk/+/android-4.2.2_r1.2/docs/system/libc/OVERVIEW.html
 // The system property named 'persist.sys.timezone' contains the name of the current timezone.
 
-const TIMEZONE_PROP_KEY: &str = "persist.sys.timezone";
+static PROPERTIES: OnceCell<AndroidSystemProperties> = OnceCell::new();
```

Comment:
Can we use a `once_cell::sync::Lazy` here? I find that a bit easier to read at call sites.

Reply:
Actually, to me that's the opposite, at least for one liner initializations. To me ↓ almost reads like a sentence:

```rust
PROPERTIES.get_or_init(AndroidSystemProperties::new)
```

Reply:
Sounds good